common/djangoapps/third_party_auth/models.py

```diff
@@ -37,6 +39,10 @@
     'username'
 ]
 
+"""
+Registers UserSocialAuth for Historical change data
+"""
+register(UserSocialAuth)
```

Comment:
The position of this feels very weird, maybe if we can find a better place to add this registration.